project/Dependencies.scala

```diff
@@ -109,8 +109,8 @@ object Dependencies {
   private val tikaV = "1.20"
   private val typesafeConfigV = "1.3.3"
   private val workbenchGoogleV = "0.15-2fc79a3"
-  private val workbenchModelV = "0.10-6800f3a"
-  private val workbenchUtilV = "0.3-f3ce961"
+  private val workbenchModelV = "0.14-27810079-SNAP"
+  private val workbenchUtilV = "0.6-27810079-SNAP"
```

Comment:
After standup I assumed you were just going to leave these unchanged. Is there method on choosing these specific versions? Should we be worried that they're still SNAPs?

Reply:
there are 3 of these, these 2 I updated b/c they weren't obviously breaking anything